Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/install-packages.md

To install a specific version of a Python-level package, include your package version in a pip [constraints file](https://pip-python3.readthedocs.io/en/latest/user_guide.html#constraints-files) and copy it into your Dockerfile. The base Docker image already has a pip constraints file which can be found [on GitHub](https://github.com/astronomer/ap-airflow/blob/master/2.1.0/buster/build-time-pip-constraints.txt).

> > **Note:** Installing dependencies will look different if you are deploying your Docker image to Astronomer via the Astro CLI. For an Astronomer platform-based setup, read [Install Packages via the Astro CLI](install-packages.md#install-packages-via-the-astronomer-cli).

Reviewer: ```suggestion
> **Note:** The process for installing dependencies is different if you are using the Astro CLI to deploy your Docker image to Astronomer. For an Astronomer platform-based setup, see [Install Packages via the Astro CLI](install-packages.md#install-packages-via-the-astronomer-cli).
```

## Install Packages via Astronomer

> If you're deploying the Astronomer Certified Docker image via Astro CLI, there are alternative workflows for installing packages and other dependencies to your image. For more information, read [Customize Images](customize-image.md).

Reviewer: ```suggestion
If you're using the Astro CLI to deploy the Astronomer Certified Docker image, there are alternative workflows for installing packages and other dependencies to your image. For more information, see [Customize Images](customize-image.md).
```
